Sources/XCDiffCore/ResultRenderer/Renderer/Renderer.swift

```diff
@@ -17,28 +17,28 @@
 import Foundation
 
 struct RendererElement {
-    enum Indent: Int {
-        case zero = 0
-        case one
-        case two
-    }
-
     enum Header: Int {
         case h1 = 1
         case h2
         case h3
     }
 
-    enum List {
-        case begin
-        case end
+    enum Style {
+        case content
+        case success
+        case warning
     }
 }
 
 protocol Renderer {
-    func text(_ text: String)
-    func list(_ element: RendererElement.List)
-    func bullet(_ text: String, indent: RendererElement.Indent)
-    func newLine(_ count: Int)
+    func begin()
```

Comment:
minor: seeing the other methods now have closure style API, is it worth having a closure style API instead of `begin()` and `end()` too?

e.g.

```swift
func render(_ content: () -> Void)
```
or 

```swift
func begin(_ content: () -> Void)
```

Reply:
I'd leave it as it's for now. We can add it once we have a use case. `Renderer` class internal so it will be no harm changing it again if needed.